src/main/java/com/puppycrawl/tools/checkstyle/utils/ModuleReflectionUtil.java

```diff
@@ -60,7 +60,6 @@ public static Set<Class<?>> getCheckstyleModules(
         return packages.stream()
                 .flatMap(pkg -> classPath.getTopLevelClasses(pkg).stream())
                 .map(ClassPath.ClassInfo::load)
-                .filter(ModuleReflectionUtil::isCheckstyleModule)
```

Comment:
I don't see this as being a good change. We only allow checkstyle modules to be picked up. We don't want a bunch of random files to be scanned. It also breaks the javadoc and method name contract. We only pull in checkstyle modules.

Reply:
yeah @rnveach , i was just trying with this change, whether i can find any regression test differences or not, i just made changes to see how it goes.

sure i do possible change if needed. once i see both regression report

Reply:
You can always put a PR into draft state, or make a statement in the first post, to show you are not ready for people to look at it.

Reply:
this mutation most likely should be covered by specific junit, not BDD styled test (with Input) as we usually do.
We have IndentationCheck that has bunch of classes around it and they are not modules.